docs/core/docker/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Docker and .NET Core
+description: Docker and .NET Core
+keywords: Docker, .NET, .NET Core
+author: spboyer
+manager: wpickett
+ms.date: 09/01/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: cae32148-aef4-4e64-a7f0-88072bad4400
+---
+
+Docker and .NET Core 
```

Comment:
nit: we're standardizing on the 

# Docker and .NET Core

syntax for headers instead of the underline.